apps/build_man/lib/build_man/file_helpers.ex

```diff
@@ -1,10 +1,8 @@
 defmodule BuildMan.FileHelpers do
   def unique_folder(prefix \\ "") do
-    system_tmp = System.tmp_dir
-    {a, b, c} = :erlang.now
-    n = node
-    dir_name = "#{prefix}#{n}-#{a}.#{b}.#{c}"
-    tmp_path = Path.join([system_tmp, "RabbitCI", dir_name])
+    dir_name = "#{prefix}#{node}-#{UUID.uuid4(:hex)}"
```

Comment:
UUIDs are globally unique so we can get rid of the node as well. I know you got rid of them, but do you think dashes in the UUID would help improve readability?